### ISSUE TITLE
Possibly lock body scrolling on mobile?

### ISSUE THREAD

**Issue by erwstout** [factor1/nifty-nav#8]
Look into the possibility of locking the page body from scrolling but allowing the navigation to scroll.

**Comment by mattada** [comment]
Are you thinking that the nav would be 100vh, and overflow-y:scroll?

**Comment by erwstout** [comment]
In mobile, yes possibly. I've had a few instances where using a fixed header/panel and the content flows out of the viewport it can be janky at times

**Comment by erwstout** [comment]
JS:

``` js
if( panelPosition == 'fixed'){
      $('body').toggleClass('nifty-locked');
    }
```

SCSS:

``` scss
.nifty-locked{
  overflow: hidden;
}

.nifty-panel{
  max-height: 100vh;
overflow-y: scroll !important;
}

```

**Comment by erwstout** [comment]
Released in `2.2.0`
